cms/djangoapps/contentstore/views/course.py

```diff
@@ -1459,32 +1459,109 @@ def _get_usage_info(store, course, split_tests):
 
             validation_summary = split_test.general_validation_message()
             usage_info[split_test.user_partition_id].append({
-                'label': '{} / {}'.format(unit.display_name, split_test.display_name),
+                'label': u"{} / {}".format(unit.display_name, split_test.display_name),
                 'url': unit_url,
                 'validation': validation_summary.to_json() if validation_summary else None,
             })
         return usage_info
 
     @staticmethod
+    def get_content_groups_usage_info(store, course):
+        """
+        Get usage information for content Group Configurations.
+        """
+        modules = store.get_items(course.id, settings={'group_access': {'$exists': True}})
```

Comment:
I am concerned about how this will perform for large courses. Has any performance testing been done?